PyInstaller/building/makespec.py

```diff
@@ -319,13 +319,11 @@ def main(scripts, name=None, onefile=None,
         # version is >= 2.4.
         try:
             import Crypto
-
-            pycrypto_version = list(map(int, Crypto.__version__.split('.')))
-            is_version_acceptable = pycrypto_version[0] >= 2 and pycrypto_version[1] >= 4
-
+            major, minor,_,_,_ = Crypto.version_info
+            temp_version = major*10+minor
+            is_version_acceptable = 24 <= temp_version
```

Comment:
If we did it this way you could run into the problem of Pycrypto version 3.0 not passing.
3 >= 2 AND 0 >=4 evaluates to True AND False which evaluates to False

If you're willing to add distutils as a dependency you can use distutils.version.looseversion to worry about this kind of thing for you.